db/fixtures/miq_product_features.yml

```diff
@@ -4612,103 +4612,117 @@
     :children:
-    - :name: View
-      :description: View Configuration Profiles
+    - :name: List
+      :description: Display Lists of Configuration Profiles
       :feature_type: view
-      :identifier: configuration_profile_view
-    - :name: Operate
-      :description: Perform Operations on Configuration Profiles
+      :identifier: configuration_profile_show_list
+    - :name: Show
+      :description: Display Individual Configuration Profile
+      :feature_type: view
+      :identifier: configuration_profile_show
+  - :name: Operate
+    :description: Perform Operations on Configuration Profiles
+    :feature_type: control
+    :identifier: configuration_profile_control
+    :children:
+    - :name: Edit Tags
+      :description: Edit Configuration Profiles Tags
       :feature_type: control
-      :identifier: configuration_profile_control
-      :children:
-      - :name: Edit Tags
-        :description: Edit Configuration Profiles Tags
-        :feature_type: control
-        :identifier: configuration_profile_tag
-  - :name: Configured Systems
-    :description: Everything under Configured Systems accordion
-    :feature_type: node
-    :identifier: configured_systems_filter_accord
+      :identifier: configuration_profile_tag
+- :name: Configured Systems
+  :description: Everything under Configured Systems
+  :feature_type: node
+  :identifier: configured_system
+  :children:
+  - :name: View
+    :description: View Configured Systems
+    :feature_type: view
+    :identifier: configured_systems_view
     :children:
-    - :name: View
-      :description: View Configured Systems
+    - :name: List
+      :description: Display Lists of Configured Systems
       :feature_type: view
-      :identifier: configured_systems_filter_accord_view
-    - :name: Operate
-      :description: Perform Operations on Configured Systems
+      :identifier: configured_system_show_list
+    - :name: Show
+      :description: Display Individual Configured System
+      :feature_type: view
+      :identifier: configured_system_show
+  - :name: Operate
+    :description: Perform Operations on Configured Systems
+    :feature_type: control
+    :identifier: configured_systems_control
+    :children:
+    - :name: Edit Tags
+      :description: Edit Configured Systems Tags
       :feature_type: control
-      :identifier: configuredsystem_control
-      :children:
-      - :name: Edit Tags
-        :description: Edit Configured Systems Tags
-        :feature_type: control
-        :identifier: configured_system_tag
-      - :name: Provision Configured Systems
-        :description: Provision Configured Systems
-        :feature_type: control
-        :identifier: configured_system_provision
+      :identifier: configured_system_tag
+    - :name: Provision Configured Systems
+      :description: Provision Configured Systems
+      :feature_type: control
+      :identifier: configured_system_provision
   - :name: Ansible Tower Job Templates
```

Comment:
These are no longer used in the UI, since we moved Ansible Tower out from configuration management. Can we get rid of this?

Reply:
@skateman actually these features are being used, but in UI this section was moved under Automation/Ansible Tower/Explorer -> Templates accordion. So i will move these features under its correct parent.

Reply:
Aren't they the `automation_manager_configuration_scripts_accord`?

Reply:
> Aren't they the `automation_manager_configuration_scripts_accord`?

this is weird/confusing, when tagging button is pressed code checks for `automation_manager_configuration_script_tag` feature https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/automation_manager_controller.rb#L57
but the toolbars on Templates screens use `configuration_script_tag`  feature https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/helpers/application_helper/toolbar/configuration_scripts/policy_mixin.rb#L10 
https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/helpers/application_helper/toolbar/configuration_script_center.rb#L24
so it looks like both feature sets are being used with a weird mix. 

Will fix this by removing `Ansible Tower Job Templates` section from `Configuration` section and changing toolbar files in UI repo to use the correct feature id.

Reply:
@skateman you can try this weird behaviour by creating a new role with features in the screenshot and then pressing any toolbar button in UI on the `Automation/Ansible Tower/Explorer/Templates` screen
![Screenshot from 2020-04-09 11-59-21](https://user-images.githubusercontent.com/3450808/78915349-c5f3f900-7a59-11ea-8e1f-82bc87f56556.png)

![Screenshot from 2020-04-09 12-00-16](https://user-images.githubusercontent.com/3450808/78915362-ca201680-7a59-11ea-9e05-e4359214d24e.png)

Reply:
Copy-paste engineering is evil :cry: